### PR TITLE
Add NSP grype airgaped usecase

### PR DESCRIPTION
**Pull request by odinnordico**
Changes in this PR are for TAP 1.6

### CODE REVIEW

File: namespace-provisioner/use-cases.hbs.md [hunk 1]
```diff
@@ -9,3 +9,5 @@ Review the following uses cases:
 [Customize OOTB default resources](use-case4.hbs.md)
 
 [Install multiple scanners in the developer namespace](use-case5.hbs.md)
+
+[Apply Scan Templates overlays from Grype package install on AirGap](use-case6.hbs.md)
```
Comment (anibmurthy) [inline namespace-provisioner/use-cases.hbs.md:13]: Would it be better to say `Template` ?

Comment (odinnordico) [reply namespace-provisioner/use-cases.hbs.md]: Changed

File: namespace-provisioner/customize-installation.hbs.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ the list of developer namespaces:
      sources. If you do not provide a `path`, Namespace Provisioner generates a `path` using `url`
      and `subPath`.
 
+>**Important** In airgap environments where a custom CA cert is needed, it is important to configure kapp-controller with the specified CA cert. This ensures that X.508/X.509 certificate errors are avoided during operation. PLease refer to [Deploying Cluster Essentials](https://{{ vars.staging_toggle }}.vmware.com/en/Cluster-Essentials-for-VMware-Tanzu/{{ vars.url_version }}/cluster-essentials/deploy.html#deploy-onto-cluster-5)
```
Comment (anibmurthy) [inline namespace-provisioner/customize-installation.hbs.md:42]: nit: typo in P**L**ease

Comment (odinnordico) [reply namespace-provisioner/customize-installation.hbs.md]: Changed

File: namespace-provisioner/use-cases.hbs.md [hunk 3]
```diff
@@ -9,3 +9,5 @@ Review the following uses cases:
 [Customize OOTB default resources](use-case4.hbs.md)
 
 [Install multiple scanners in the developer namespace](use-case5.hbs.md)
+
+[Apply Scan Template overlays from Grype package install on AirGap](use-case6.hbs.md)
```
Comment (atmandhol) [inline namespace-provisioner/use-cases.hbs.md:13]: *suggestion*: `Apply scanner ScanTemplate overlays on Air-gapped environments.

File: toc.hbs.md [hunk 4]
```diff
@@ -470,6 +470,7 @@ docs.vmware.com is built.
           - [Work with Private Git Repositories](namespace-provisioner/use-case3.hbs.md)
           - [Customize default resources](namespace-provisioner/use-case4.hbs.md)
           - [Install multiple scanners in the developer namespace](namespace-provisioner/use-case5.hbs.md)
+          - [Apply Scan Template overlays from Grype package install on AirGap](namespace-provisioner/use-case6.hbs.md)
```
Comment (atmandhol) [inline toc.hbs.md:473]: **suggestion**: Apply scanner ScanTemplate overlays on Air-gapped environments

File: namespace-provisioner/use-case6.hbs.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+# Apply Scan Template overlays from Grype package install on AirGap
```
Comment (atmandhol) [inline namespace-provisioner/use-case6.hbs.md:1]: **suggestion**: Apply scanner ScanTemplate overlays on Air-gapped environments

File: namespace-provisioner/use-case6.hbs.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+The namespace provisioner includes pre-configured `grype-scanner` PackageInstall for each developer namespace, as described in the [Default resources]((namespace-provisioner/default-resources.hbs.md)) section of the documentation. If you require customization of the Scan Template created by the PackageInstall, it is important to note that the namespace provisioner does not directly create the Scan Template. Therefore, overlays should be applied to the Scan Template through package customization (For more information on how to customize a package installation, please refer to [Customize a package](customize-package-installation.hbs.md)).
```
Comment (atmandhol) [inline namespace-provisioner/use-case6.hbs.md:3]: `ScanTemplate` - one word as that is a CR

Comment (atmandhol) [inline namespace-provisioner/use-case6.hbs.md:3]: Point people to this link https://github.com/pivotal/docs-tap/blob/main/install-offline/grype-offline-airgap.hbs.md where all the possible customizations that they can do on `grype` in air-gapped environment is listed

Comment (atmandhol) [reply namespace-provisioner/use-case6.hbs.md:3]: `Customize a package` - points to a page that does not exist

Comment (atmandhol) [inline namespace-provisioner/use-case6.hbs.md:3]: `Customize a package` - points to a page that does not exist

Comment (odinnordico) [reply namespace-provisioner/use-case6.hbs.md]: The page does exist: https://github.com/pivotal/docs-tap/blob/main/customize-package-installation.hbs.md

Comment (odinnordico) [reply namespace-provisioner/use-case6.hbs.md]: Added a phrase with that

Comment (odinnordico) [reply namespace-provisioner/use-case6.hbs.md]: Changed all usages

File: namespace-provisioner/use-case6.hbs.md (hunk 5 above)
Comment (odinnordico) [reply namespace-provisioner/use-case6.hbs.md]: Applied

File: toc.hbs.md (hunk 4 above)
Comment (odinnordico) [reply toc.hbs.md]: Applied

File: namespace-provisioner/use-cases.hbs.md (hunk 3 above)
Comment (odinnordico) [reply namespace-provisioner/use-cases.hbs.md]: Applied

**Review by atmandhol — APPROVED** [review]

**Comment by atmandhol** [comment]
@kcoriordan please review and merge. Thank you!

**Pull request merged**